src/gsi_varbc/satbias_upgrader.py

```diff
@@ -52,19 +55,39 @@ def satbias_upgrader(infile, outfile):
         bias_coeff = oldnc.variables['bias_coefficients'][:]
         for i, pred in enumerate(predictors):
             temp = pred.split('_')
-            predOut = temp[0] + ''.join(ele.title() for ele in temp[1:])
-            if predOut == 'emissivity':
-                predOut = 'emissivityK'
+            if "order" in temp:
+                idorder = temp.index('order')
+                predOut = temp[0] + ''.join(ele.title() for ele in temp[1:idorder])+'_'+temp[idorder]+'_'+temp[idorder+1]
+            else:
+                predOut = temp[0] + ''.join(ele.title() for ele in temp[1:])
```

Comment:
This is fine but we could use a dictionary instead of all these `if` statements

Reply:
True. It was added on the fly one after another when I was updating the variable names. Let's me try to update these with a dictionary.

Reply:
I used dictionary to replace all the ifs except the "order" part. I don't know whether there is a different way to do this. 
```
            idorder = temp.index('order') if "order" in temp else None
            predOut = temp[0] + ''.join(ele.title() for ele in temp[1:idorder]) + \
                  ('_' + temp[idorder] + '_' + temp[idorder + 1] if idorder is not None else '')

            # Replace strings using dictionary
            for key, value in replace_dict.items():
                if key in predOut:
                    predOut = predOut.replace(key, value)
```